src/radiation/cooling_function.cpp

```diff
@@ -179,14 +189,24 @@ TaskStatus CoolingFunctionCalculateFourForce(MeshBlockData<Real> *rc, const doub
     const parthenon::AllReduce<bool> *pdo_gain_reducer =
         rad->MutableParam<parthenon::AllReduce<bool>>("do_gain_reducer");
     const bool do_gain = pdo_gain_reducer->val;
+
+    // Code to CGS
+    const Real density_conversion_factor = unit_conv.GetMassDensityCodeToCGS();
+    const Real temperature_conversion_factor = unit_conv.GetTemperatureCodeToCGS();
+    const Real length_conversion_factor = unit_conv.GetLengthCodeToCGS();
+
+    // CGS to code
+    const Real energy_conversion_factor = unit_conv.GetEnergyCGSToCode();
+    const Real mass_conversion_factor = unit_conv.GetMassCGSToCode();
+    const Real time_conversion_factor = unit_conv.GetTimeCGSToCode();
```

Comment:
These lines probably don't need to be here, because the same definitions are above in the same function.